ExpandingMenu/Classes/ExpandingMenuButton.swift

```diff
@@ -69,62 +76,57 @@ open class ExpandingMenuButton: UIView, UIGestureRecognizerDelegate {
     }
     
     @objc open var bottomViewAlpha: CGFloat = 0.618
-    
     @objc open var titleTappedActionEnabled: Bool = true
-    
-    open var expandingDirection: ExpandingDirection = ExpandingDirection.top
-    open var menuTitleDirection: MenuTitleDirection = MenuTitleDirection.left
-    
-    open var enabledExpandingAnimations: CustomAnimationOptions = .Default
-    open var enabledFoldingAnimations: CustomAnimationOptions = .Default
-    
+    @objc open var expandingDirection: ExpandingDirection = .top
+    @objc open var menuTitleDirection: MenuTitleDirection = .left
+    open var expandingAnimations: CustomAnimationOptions = [.move, .menuButtonRotate]
```

Comment:
Please use `[.default]` or please describe the reason why this value should be `[.move, .menuButtonRotate]`.